### ISSUE TITLE
Positon Right + Page Shadow not working

### ISSUE THREAD

**Issue by jrean** [FrDH/jQuery.mmenu#857]
Hi,

`"jquery.mmenu": "^7.0.1"`

```
    'node_modules/jquery.mmenu/dist/jquery.mmenu.css',
    'node_modules/jquery.mmenu/dist/extensions/positioning/jquery.mmenu.positioning.css',
    'node_modules/jquery.mmenu/dist/extensions/shadows/jquery.mmenu.shadows.css',
```

- When using default behaviour (menu on the left) + page shadow, everything is working like a charm.

```
    // Mmenu
    $("#nav-menu").mmenu({
        extensions: [
            "shadow-page",
            "position-right",
        ]
    }, {
        offCanvas: {
            pageSelector: "#page-wrapper"
        }
    });

    var API = $("#nav-menu").data("mmenu");

    $(".toggle-nav-menu").click(function() {
        API.open();
    });
```

- When setting the right position, then the shadow is not visible anymore (but still present). It is hidden by the page sliding to the left...

Can't find any solution and the documentation is not mentioning anything about it.

http://mmenu.frebsite.nl/documentation/extensions/shadows.html
Documentation mention about: `<link href="path/to/jquery.mmenu.pageshadow.css" rel="stylesheet" />`. If I'm not wrong it is: `path/to/jquery.mmenu.shadows.css`

Thanks for the help

**Comment by FrDH** [comment]
Thank you for bringing this issue to my attention, I'll check and fix the issue.

**Comment by jrean** [comment]
Awesome!
Thxs

**Comment by redpik** [comment]
and shadow-panels doesn't work because of `.mm-panels` overflow hidden.

**Comment by FrDH** [comment]
Shadow-panels works as intended, its just that the shadow is only visible while the panels shift. Try using it in combination with the [icon panels add-on](http://mmenu.frebsite.nl/documentation/addons/icon-panels.html).

**Comment by FrDH** [comment]
page shadow bug should be fixed in 7.0.2
